src/ducks/reimbursements/HealthReimbursements.styl

```diff
@@ -13,3 +13,8 @@
 
     +small-screen()
         margin-top 16px
+
+.Caption
+    font-size 14px
+    line-height 1.7
+    color var(--coolGrey)
```

Comment:
@gooz do we have already a Text class for this ?

Reply:
We do but the font-size is 12px and the line-height is 1.2

Reply:
I didn't find a Text class with these styles. I asked the design team if it's a new style of text that we should put in cozy-ui. I think we can merge as is and refactor when we have their answer.